.env

```diff
@@ -1,39 +1,38 @@
-## you should add the environment variable in .zshrc or other profile on production enviroment
-
-## Database connect and pool configuration
-MEGA_DB_POSTGRESQL_URL = "postgres://${PG_USERNAME}:${PG_SECRET}@${PG_HOST}/mega"
-MEGA_DB_MYSQL_URL = "mysql://${MYSQL_USERNAME}:${MYSQL_SECRET}@${MYSQL_HOST}/mega"
-
+# If you followed the installation guide, you can use below URL directly, comment it the otherwise.
+MEGA_DB_POSTGRESQL_URL = "postgres://mega:rustgit@127.0.0.1/mega"

```

Comment:
Do we need to display the username and password in plain text? I recommend configuring them in production as environment variables, while using plaintext for local development